bin/gltf-pipeline.js

```diff
@@ -78,6 +79,7 @@ fs.readFile(gltfPath, function (err, data) {
 
         combineMeshes(gltf);
         combinePrimitives(gltf);
+        combineNodes(gltf);
```

Comment:
Are we sure that we don't want to call this before `combineMeshes` and `combinePrimitives`?  Or does `combineNodes` also explicitly combine its meshes/primitives?